src/convertToObject.js

```diff
@@ -5,8 +5,25 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(str) {
+  const resultStyles = {};
+  const listStyle = str.split(';');
+
+  for (const listStyleElement of listStyle) {
+    const styleItem = listStyleElement.trim();
+
+    if (styleItem) {
+      const [property, ...itemParts] = styleItem.split(':');
```

Comment:
Destructuring with '...' to capture the rest of the parts is good, but you should also consider trimming the property before checking if it exists to handle cases where there may be whitespace before the colon.